Source/JavaScriptCore/inspector/protocol/Canvas.json

```diff
@@ -23,7 +23,12 @@
         {
             "id": "ContextType",
             "type": "string",
-            "enum": ["canvas-2d", "offscreen-canvas-2d", "bitmaprenderer", "webgl", "webgl2"],
+            "enum": [
+                "canvas-2d", "offscreen-canvas-2d",
```

Comment:
Makes me wonder if "offscreen" should just be another boolean instead of a context type? (Better to do so know when we only have to shim compatibility for a single existing value)